cmd/teleirc.go

```diff
@@ -1,16 +1,45 @@
+// Package main contains all logic relating to running TeleIRC
 package main
 
 import (
+	"flag"
 	"fmt"
-
 	"github.com/ritlug/teleirc/internal"
+	"github.com/ritlug/teleirc/internal/handlers/irc"
+	tg "github.com/ritlug/teleirc/internal/handlers/telegram"
 )
 
+func startIrc() {
+	ircbot, err := irc.StartBot()
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Println(ircbot)
+}
+
+func startTelegram() {
+	tgbot, err := tg.StartBot()
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Println(tgbot)
+}
+
 func main() {
-	settings, err := internal.LoadConfig("../env.example")
+	// optional path flag if user does not want .env file in root dir
+	var path string
+	flag.StringVar(&path, "p", "../.env", "Path to .env")
```

Comment:
I think setting the default value to ".env" might make more sense, given that we're usually going to specify our own env file during development.

Reply:
yeah that seems to be my general thought as well.